### DISCUSSION TITLE
Donate

### DISCUSSION THREAD

**Discussion by jupitert25** [engageub/InternetIncome#100] [Ideas]
Should add Donate link on README

**Comment by engageub** [comment]
I had considered it earlier, but I was concerned that people might think this script is not generating income if I were to ask for donations. Consequently, I decided to remove the donation request. It is now located inside a hidden folder named '.github' in the code.

Thank you

**Reply by jupitert25** [reply]
I'm not sure I understand what's in there. so just replace it with "engageub" username?

**Reply by jupitert25** [reply]
it's because of how you support everyone with a free script. if they make some they can donate some for you to make the script better.

**Reply by engageub** [reply]
Just to let you know, if its mentioned patreon and a user or email address is present beside it, you have to visit patreon website and search with that username or email.

**Reply by engageub** [reply]
Thank you for considering a donation, but please note that donations will not make script better.
Users have been reporting issues through "Issues" tab if they come across something. 
Currently, the script is running smoothly without any reported issues.

**Reply by jupitert25** [reply]
BTW honeygain new version working now?

**Reply by engageub** [reply]
It shows waiting for traffic on start for sometime and then works normally. I am using it with direct connection.

**Reply by engageub** [reply]
Also, while using honeygain in different folders, you have to change the device name if you are using same account else account may be blocked. If you are running all 10 from same folder, different device names are automatically created.

**Reply by jupitert25** [reply]
Already done that but I kinda of want to switch to the test version but only way to do that is to start all over .

**Reply by engageub** [reply]
As mentioned, its a one time setup. You have use --delete to delete the existing ones and add new application in properties.conf and use --start to start it.
It will be become complex if more features are added and new users may find it difficult setting up.

**Reply by engageub** [reply]
The test branch has more network options and it is mandatory to specify the network options without which the applications do not start.

**Comment by engageub** [comment]
That should be fine. You may use the referral links mentioned to sign up.
That way you will not lose anything.

**Reply by jupitert25** [reply]
earnapp ban wave on the user running many devices with docker, since they don't support docker so can't say anything about that. Very sad

**Reply by jupitert25** [reply]
I wonder Is LXC allowed

**Reply by engageub** [reply]
Hello,
Regarding earnapp, you will have to delete the nodes which are detected as Proxy/VPN. 
Docker creates an isolated environment.
LXC containers also create separate containers. 
Its the IP that's being considered. You can share your home internet connection and use it as proxy on other host.
The traffic goes through your home IP connection which is residential. 

Thank you

**Reply by jupitert25** [reply]
kinda weird that Earnapp never allows proxy or vpn when it is detected it will show warning devices.  some got banned in their discord and said they running docker. Admin said docker unofficial on their site and it's true . but it seems like the only account that runs many devices or earns too much.

**Reply by engageub** [reply]
Yes, it may be due to the number of devices. To determine which data can be visible to Earnapp, you can visit the Bitping node dashboard and click on the nodes. You will see the CPU, Linux version, and RAM. That's what Earnapp usually sees if they obtain information while the container is isolated. It's similar to a cloud VPS running on a physical machine. On a single physical machine, multiple cloud VPS or virtual machines can be deployed. Similarly, multiple isolated containers can be deployed on a single physical machine. The data goes through the network, and anything can be modified before it goes out of the network. It requires expertise.
Official apps are being used in the docker images, although docker images are not officially published.

Thank you

**Reply by jupitert25** [reply]
I got ban all 2 accounts with over 30 devices. oh well i guess have to use other app

**Reply by engageub** [reply]
Are your proxies considered residential? Also, are they from same region or different regions.
They are doing prechecks not sure why accounts are banned. 


Thank you

**Reply by engageub** [reply]
![image](https://github.com/engageub/InternetIncome/assets/103608400/4b8fd56d-502e-48a6-bdbb-7990e5976a35)

**Reply by jupitert25** [reply]
all are residential and same region. yes after the release that message a lot people got ban and me too

**Reply by jupitert25** [reply]
![image](https://github.com/engageub/InternetIncome/assets/151444646/9cb8ea5e-ed24-45f0-bbfb-4a56b0d13fa7)

**Reply by jupitert25** [reply]
![image](https://github.com/engageub/InternetIncome/assets/151444646/13696c4d-d78e-42a5-acb2-da147c1bc9d8)

**Reply by jupitert25** [reply]
still don't know how they can check who using docker

**Reply by engageub** [reply]
May be you have 15 devices online and that may be the reason why it was flagged or some other user may have used the same UUID that you already have. 

Thank you

**Comment by engageub** [comment]
Closing this discussion as topic related to donation has been answered.
